packages/replay/src/bin.ts

```diff
@@ -72,6 +73,7 @@ program
     "--api-key <key>",
     "Authentication API Key"
   )
+  .option('-a, --all', 'Upload all recordings')
```

Comment:
This is a bit nitty but I think `npx replay upload-all --all` is awkward. Maybe `--include-in-progress` or `--include startedWrite,startedUpload` or something else?

Reply:
or maybe `--status all|onDisk|crashed|startedWrite|startedUpload`?

Reply:
I wasn't sure what the use case is for individually enumerating the status(es) the user might be filtering for, so kept it simple with  `--include-in-progress` for now

Reply:
I'm not sure either. I don't want to change APIs so I was offered it as a flexible option. I'm okay with your choice for now and we can drop it in v2 if we decide to go a different route.